Add public_network_access_enabled property to app service docs (#22399)
hashicorp/terraform-provider-azurerm: website/docs/r/windows_web_app.html.markdown

* `https_only` - (Optional) Should the Windows Web App require HTTPS connections.

> * `public_network_access_enabled` - (Optional) Whether or not public network access is enabled for the Windows Web App. Defaults to `true`.

Reviewer: ```suggestion
* `public_network_access_enabled` - (Optional) Should public network access be enabled for the Function App. Defaults to `true`.
```
